rag/advanced_rag/harness/orchestrator/decompose.py

```
@@ -0,0 +1,111 @@
+        ctx.iteration = cycle
+        unverified = [c for c in ctx.claims if not c.is_verified]
+        if not unverified:
+            break
+
+        # Parallel search on unverified claims
+        tasks = []
+        for c in unverified:
+            tasks.append(hybrid_search(tools, query=c.description))
+        results = await asyncio.gather(*tasks)
+
+        for c, result in zip(unverified, results):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)